es_find_block.py

```diff
@@ -0,0 +1,45 @@
+from .es_base_block import ESBase, Limitable, Sortable
+from nio.common.discovery import Discoverable, DiscoverableType
+from nio.metadata.properties.expression import ExpressionProperty
+from nio.common.signal.base import Signal
+
+
+@Discoverable(DiscoverableType.block)
+class ESFind(Limitable, Sortable, ESBase):
```

Comment:
This is probably the block where we want the search command, if we even need the command anymore...